### ISSUE TITLE
Animation problems, the body cannot move.

### ISSUE THREAD

**Issue by Misterkiwi500** [guoyww/AnimateDiff#75]
I noticed that the stable diffusion webui had problems because of animation. that's so, the body can't move freely and that's why i prepared for him.

8 fps:
![df14f4db-02e8-4a13-a062-ca82b8018f5a](https://github.com/guoyww/AnimateDiff/assets/53793601/2f374757-462a-4969-8b25-3f082cf9cf47)

32 fps:
![01d0ef05-98cb-4c4b-b30d-debc89dbf82c](https://github.com/guoyww/AnimateDiff/assets/53793601/6b54c6a0-6a2f-4f78-9ae2-e2422ebb2820)

128 fps.
![95018262-537b-49f1-b309-adab32c7745b](https://github.com/guoyww/AnimateDiff/assets/53793601/4de03fbb-72c6-4a6f-986f-1e2f55056dd5)

256 fps: 
![14e76947-324a-400c-9f94-77b9689a8e65](https://github.com/guoyww/AnimateDiff/assets/53793601/fba0a836-1ab9-44d5-b72c-bc43c4251c2a)

I think maybe it's because the two models are wrong, because of mm_sd_v14.ckpt and mm_sd_v15.ckpt 

oh my, what are we doing? I don't know!

**Comment by CiaoHe** [comment]
+1,I discovered that using the mm_sd_v14.ckpt can significantly enhance the performance of 1-ToonYou (compare to use `mm_sd_v15.ckpt`) btw, I use `stable-diffusion-v1-5` as base-model 
![sample-min](https://github.com/guoyww/AnimateDiff/assets/39040787/a4268ced-9d8c-4930-91df-c22d2fb2c6b6)
(Top: mm_sd_v14.ckpt; Bottom:mm_sd_v15.ckpt)

**Comment by Mistermango23** [comment]
Thank!

**Comment by Mistermango23** [comment]
Thanks"

**Comment by Bendito999** [comment]
Regarding the talesofai initial image repo (and probably also this repo, but not necessarily the Auto1111 repo), regarding the amount of movement the picture has, if you are using a lora that is heavily tuned to 1 image that you are trying to use as an initial image also, you can get more movement by setting the lora alpha to be lower instead of higher (at the cost of the AI getting too creative and varying the picture more). 
Lora alpha 1.0  = very little movement, almost static
Lora alpha 0.8 = tiny bit of movement
Lora alpha 0.6 = significant movement while still maintaining initial picture
Lora alpha 0.5 = a little bit more movement still, but some seeds lose a little too much of the picture. 
Lora alpha 0.4 = lots of movement, but AI sometimes getting too creative and losing the initial picture. 

Basically just a note to check on how high you have Lora alpha set in the .yaml (or Gradio) when using Lora, if it is set a 0.8 and you want more movement, putting it at 0.6 can increase movement, based on tests I've done on a non ai generated art picture that I've been trying to get this to animate.

**Comment by haofanwang** [comment]
The default alpha is 0.6 already. There mush be something wrong.
